src/lib/storage/storage.ts

```diff
@@ -2,7 +2,7 @@ import {
   createConnection,
   ConnectionOptions,
   Connection,
-} from 'typeorm/browser'
```

Comment:
This one especially, importing `createConnection` from `typeorm` instead of `typeorm/browser`, I think it might cause problems with react-native